charts/thoras/templates/collector/deployment.yaml

```diff
@@ -36,10 +36,12 @@ spec:
         command: ["/bin/sh", "-c"]
         args:
           - |
-            chown -R 1000:0 /usr/share/elasticsearch/data
-            rm -f /usr/share/elasticsearch/data/node.lock
+            mkdir -p /var/lib/share/elasticsearch/data
+            find /var/lib/share/elasticsearch -not -name postgresql -not -name data -mindepth 1 -maxdepth 1 -exec mv {} /var/lib/share/elasticsearch/data \;
```

Comment:
I'm sure I'm just not fully understanding , but if the `find` command here is meant to move the files out of the legacy filepath (`/usr/share/elasticsearch/`), shouldn't that be the first argument to `find`? It looks like it's finding in the directory it just created in the line above and not the old directory path

Reply:
The `find` command is relative to the mount point, which also changed.

Aside: the mount point moved from `/usr/share/elasticsearch` because that's actually where elastic is mounting it's other files, including `bin`.